sdk/purview/purview-workflow-rest/test/public/userrequest.spec.ts

```diff
@@ -25,7 +25,7 @@ describe("Submit a user request", () => {
   });
 
   it("should submit a user request.", async function () {
-    const options: SubmitUserRequestsParameters = {
+    const options: UserRequestsSubmitParameters = {
```

Comment:
Our testing recordings are already upgraded to the new asset repo. You have to run your test in record mode and then run `npx dev-tool test-proxy push` to push recordings so that the ci could pass (see [more](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/ASSET_SYNC_MIGRATION.md#workflow-with-asset-sync-enabled)).

```
$ export TEST_MODE=record && rushx test
$ npx dev-tool test-proxy push
```

Reply:
Thanks pretty much for the help!! But `rushx test` should fail because I still cannot fix this issue with `unit-test`. When I run `rushx unit-test -o @azure-rest/purview-workflow` alone, it reports: 

>  rush unit-test -o  @azure-rest/purview-workflow
> Found configuration in C:\azure-sdk-for-js\rush.json
> 
> 
> Rush Multi-Project Build Tool 5.106.0 - https://rushjs.io
> Node.js version is 18.18.2 (LTS)
> 
> Found configuration in C:\azure-sdk-for-js\rush.json
> 
> Starting "rush unit-test"
> 
> Analyzing repo state... DONE (0.50 seconds)
> 
> Executing a maximum of 15 simultaneous processes...
> 
> ==[ @azure-rest/purview-workflow ]=================================[ 1 of 1 ]==
> C:\azure-sdk-for-js\common\tools\dev-tool\src\util\testProxyUtils.ts:167
>           reject(new Error(`${executable} exited with code ${exitCode}`));
>                  ^
> Error: C:\Users\weiranwu\AppData\Local\azsdk-dev-tool-nodejs\Cache\test-proxy\1.0.0-dev.20230912.4\Azure.Sdk.Tools.TestProxy.exe exited with code 1
>     at ChildProcess.<anonymous> (C:\azure-sdk-for-js\common\tools\dev-tool\src\util\testProxyUtils.ts:167:18)
>     at ChildProcess.emit (node:events:517:28)
>     at ChildProcess.emit (node:domain:489:12)
>     at Process.ChildProcess._handle.onexit (node:internal/child_process:292:12)
> Returned error code: 1
> "@azure-rest/purview-workflow" failed to build.
> 
> 
> 
> ==[ FAILURE: 1 operation ]=====================================================
> 
> --[ FAILURE: @azure-rest/purview-workflow ]------------------[ 2.52 seconds ]--
> 
> C:\azure-sdk-for-js\common\tools\dev-tool\src\util\testProxyUtils.ts:167
>           reject(new Error(`${executable} exited with code ${exitCode}`));
>                  ^
> Error: C:\Users\weiranwu\AppData\Local\azsdk-dev-tool-nodejs\Cache\test-proxy\1.0.0-dev.20230912.4\Azure.Sdk.Tools.TestProxy.exe exited with code 1
>     at ChildProcess.<anonymous> (C:\azure-sdk-for-js\common\tools\dev-tool\src\util\testProxyUtils.ts:167:18)
>     at ChildProcess.emit (node:events:517:28)
>     at ChildProcess.emit (node:domain:489:12)
>     at Process.ChildProcess._handle.onexit (node:internal/child_process:292:12)
> 
> 
> Operations failed.
> 
> rush unit-test (3.32 seconds)

However, `rushx build -o @azure-rest/purview-workflow`, `rushx build:test -o @azure-rest/purview-workflow` work just fine. I'm still trying to fix this. Have you ever seen this error before?